nemo_text_processing/text_normalization/zh/data/measure/units_en.tsv

```diff
@@ -40,23 +38,6 @@ kw	千瓦
 kW	千瓦
 lb	磅
 lbs	磅
-m2	平方米
-m²	平方米
-m3	立方米
-m³	立方米
-mbps	兆比特每秒
-mg	毫克
-mhz	兆赫兹
-mi2	平方英里
-mi²	平方英里
-mi	英里
-min	分钟哦
-ml	毫升
-mm2	平方毫米
-mm²	平方毫米
-mol	摩尔
-mpa	兆帕
-mph	英里每小时
```

Comment:
why these are removed?

Reply:
Hi Evelina, these are included in the grammar for measure. I found out when running test, some of the symbols get separated. For example, "mm²" here will get treated a "m" and "m²" separately. I had to make one-to-one mapping in the tagger grammar to cover them.